data_summaries/Cidar.json

```diff
@@ -0,0 +1,32 @@
+{
+  "dataset-name (must match unique identifier)": {
+    "Unique Dataset Identifier": "cidar",
+    "Dataset Name": "cidar",
+    "Paper Title": "CIDAR: Culturally Relevant Instruction Dataset For Arabic",
+    "Dataset URL": "https://github.com/ARBML/CIDAR",
+    "GitHub URL": "https://github.com/ARBML/CIDAR",
+    "Hugging Face URL": "https://hf.co/datasets/arbml/CIDAR",
+    "Papers with Code URL": "https://paperswithcode.com/dataset/cidar",
+    "ArXiv URL": "https://arxiv.org/abs/2402.03177",
+    "Semantic Scholar Corpus ID": "",
+    "Collection": "Cidar",
+    "Collection URL": "https://hf.co/datasets/arbml/CIDAR",
+    "Languages": ["Arabic"],
+    "Task Categories": ["Instruction Generation"],
```

Comment:
Can you copy and paste the task categories from the data_summaries/alpaca.json, since this is largely derived from that? And then maybe suggest 1-2 from the AskTheTeacher website?